jobs/tendrl-cluster-test-jobs.yml

```diff
@@ -0,0 +1,322 @@
+
+
+      - shell: |
+          #!/bin/bash -x
+          grep -q ceph ${{HOME}}/cluster-{clustername}.hosts || exit 0
+          qe_server=$(grep -A1 '\[qe_server\]' ${{HOME}}/cluster-{clustername}.hosts | tail -1)
+          ssh usmqe@${{qe_server}} "cd /home/usmqe/ceph-ansible; \
+            ANSIBLE_CONFIG=/tmp/ansible.cfg \
+            ANSIBLE_ROLES_PATH=/home/usmqe/ceph-ansible/roles \
+            ANSIBLE_FORCE_COLOR=true \
+            ansible-playbook -i /home/usmqe/cluster-{clustername}.hosts -vvv \
+            -u root /home/usmqe/ceph-ansible/site.yml"
+
+    publishers:
+      - trigger-parameterized-builds:
+          - project: 'tendrl-2-2-cluster-{clustername}-test-api'
+            current-parameters: true
+            node-parameters: true
+            condition: SUCCESS
+
+      - trigger-parameterized-builds:
+          - project: 'tendrl-2-X-cluster-{clustername}-teardown'
+            current-parameters: true
+            node-parameters: true
+            condition: FAILED
+
+
+- job-template:
+    name: "tendrl-2-2-cluster-{clustername}-test-api"
+    display-name: 'Tendrl-6: cluster-{clustername} Test API'
+    description: 'Do not edit this job through the web!'
+    scm:
+      - git:
+          url: 'https://github.com/Tendrl/usmqe-tests.git'
+          branches:
+            - "*/master"
+          basedir: usmqe-tests
+          skip-tag: true
+          wipe-workspace: true
+    project-type: freestyle
+    defaults: global
+    parameters:
+      - string:
+          name: TEST_LIST
+          default: others,gluster
+          description: "List of tests to execute (and any additional parameter for py.test)."
+    disabled: false
+    triggers:
+      - timed: ''
+    concurrent: false
+    quiet-period: 5
+    block-downstream: true
+    block-upstream: true
+    wrappers:
+      - ansicolor:
+          colormap: "xterm"
+    node: '{node}'
+    builders:
+      - shell: |
+          #!/bin/bash -x
+          #cd usmqe-tests
+          ## prepare hosts file
+          qe_server=$(grep -A1 '\[qe_server\]' ${{HOME}}/cluster-{clustername}.hosts | tail -1)
+          scp ${{HOME}}/cluster-{clustername}.hosts \
+            usmqe@${{qe_server}}:/home/usmqe/usmqe-tests/conf/usm.hosts
+          scp ${{HOME}}/cluster-{clustername}.ini \
+            usmqe@${{qe_server}}:/home/usmqe/usmqe-tests/conf/usm.ini
+          # prepare required variables
+          tests=""
+          for test in ${{TEST_LIST//[_,]/ }}; do
+            tests="${{tests}} usmqe_tests/api/tendrl/${{test}}";
+          done
+          # prepare env
+          ssh usmqe@${{qe_server}} "export PATH=\$PATH:\$HOME/.local/bin:\$HOME/bin; \
+            scl enable rh-python35 \
+              'pip install --user -r /home/usmqe/usmqe-tests/requirements.txt' ; \
+            source /opt/rh/rh-python35/enable ; \
+            export X_SCLS=\$(scl enable rh-python35 'echo $X_SCLS') ; \
```

Comment:
I don't think enabling python35 softcol manually here is necessary, as `qe-server-user` role enables it in bashrc file of usmqe user.